Resources/views/Tree/tree.html.twig

```diff
@@ -46,7 +88,9 @@ file that was distributed with this source code.
                     label: "{{ 'delete_item'|trans({}, 'SonataDoctrinePhpcrAdminBundle') }}",
                     icon: 'fa fa-trash-o'
                 }
-            }
+            },
+            dnd: {enabled: true, reorder: true},
+            sortableBy: 'position'
```

Comment:
@wouterj  is it ok to have a single option for that on tree implementation? or just into dnd section?